### PR TITLE
Cleanup: fix variable names in comments

### PR DESCRIPTION
**Pull request by bzsuni**
What type of PR is this?
/kind cleanup
What this PR does / why we need it:
optimised some code, such as the comment does not match the exported var

### CODE REVIEW

File: pkg/util/iptables/testing/fake.go [hunk 1]
```diff
@@ -271,12 +271,11 @@ func (f *FakeIPTables) restoreTable(newDump *IPTablesDump, newTable *Table, flus
...
-				} else {
-					// rule is a new rule that jumped to a chain that was
-					// neither created nor pre-existing
-					oldTable.Chains = backupChains
-					return fmt.Errorf("rule %q jumps to a non-existent chain", rule.Raw)
 				}
+				// rule is a new rule that jumped to a chain that was
+				// neither created nor pre-existing
+				oldTable.Chains = backupChains
+				return fmt.Errorf("rule %q jumps to a non-existent chain", rule.Raw)
```
Comment (danwinship) [inline pkg/util/iptables/testing/fake.go:278]: I disagree with this change (and the similar changes to `pkg/util/iptables/testing/parse.go`; this isn't "if special case, do X, else do the normal thing". It's "There are exactly two possibilities; if it's the first possibility, do the first thing, else it's the second possibility, so do the second thing." Using an `else` represents the logical structure better in this case IMO.

Comment (aojea) [reply pkg/util/iptables/testing/fake.go:278]: is a "go thing" as you can read it here https://go.dev/doc/effective_go#if

> In the Go libraries, you'll find that when an if statement doesn't flow into the next statement—that is, the body ends in break, continue, goto, or return—the unnecessary else is omitted.

Comment (bzsuni) [reply pkg/util/iptables/testing/fake.go]: > I disagree with this change (and the similar changes to `pkg/util/iptables/testing/parse.go`; this isn't "if special case, do X, else do the normal thing". It's "There are exactly two possibilities; if it's the first possibility, do the first thing, else it's the second possibility, so do the second thing." Using an `else` represents the logical structure better in this case IMO.

I have canceled such changes

Comment (danwinship) [reply pkg/util/iptables/testing/fake.go]: > is a "go thing" as you can read it here https://go.dev/doc/effective_go#if

They don't distinguish "if special case, do X, else do the normal thing" from "if case 1 do thing 1 else do thing 2" in the recommendation, but the code examples they give are all the former, not the latter.

Comment (bzsuni) [reply pkg/util/iptables/testing/fake.go]: what you mean is to keep the previous changes?

**Comment by danwinship** [comment]
no longer touches any sig-network files
/remove-sig network
/uncc
/auto-cc
/ok-to-test

**Comment by alculquicondor** [comment]
/retitle Cleanup: fix variable names in comments
/kind cleanup

For future reference, optimizing generally refers to improving the run time, memory or some other metric.

/lgtm
/approve

**Pull request merged**